arch/x86/x86_64/include/uk/asm/traps.h

```diff
@@ -63,3 +63,5 @@ struct ukarch_trap_ctx {
 #define UKARCH_TRAP_MATH		trap_math
 
 #define UKARCH_TRAP_SECURITY		trap_security
+
+#define UKARCH_TRAP_X86_GP		trap_gp_fault
```

Comment:
The name contains the `X86` because the event is not available on any other architecture. The other events are meant to be generic and available on all architectures.